exercises/sum-of-multiples/example.rb

```diff
@@ -1,6 +1,6 @@
 class SumOfMultiples
   def self.to(limit)
```

Comment:
this method could be removed if desired I guess, since it's not being used in the tests anymore